src/index.html

```diff
@@ -10,13 +10,104 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
+    <title>Moyo header</title>
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          class="logo"
+          src="/src/images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="selectors">
```

Comment:
Fix classes everywhere
```suggestion
          <li class="nav_item">
```